zipline/pipeline/loaders/blaze/core.py

```diff
@@ -194,6 +195,7 @@
 )
 is_invalid_deltas_node = complement(flip(isinstance, valid_deltas_node_types))
 get__name__ = op.attrgetter('__name__')
+log = logbook.Logger('BlazeLoader')
```

Comment:
see comment below about the log; however, can we name loggers with `__name__` to make it easier to find the source of the logging?